### PR TITLE
Add support for changing intensity of LEDs

### PR DESCRIPTION
**Pull request by kdarkhan**
I implemented the intensity functionality by right shifting each component of RGB color.
Intensities range range from 0 to 3. Default intensity is 0 which corresponds to the original color.
Corresponding PR in the main repo is [here](https://github.com/OpenAnnePro/qmk_firmware/pull/15/files)

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -26,25 +26,26 @@
 static void columnCallback(GPTDriver* driver);
 static void animationCallback(GPTDriver* driver);
 static void executeMsg(msg_t msg);
-static void switchProfile(void);
```
Comment (kdarkhan) [inline main.c:29]: removed unused function

File: main.c [hunk 2]
```diff
@@ -26,25 +26,26 @@
...
-  LINE_LED_COL_1, 
-  LINE_LED_COL_2, 
-  LINE_LED_COL_3, 
-  LINE_LED_COL_4, 
-  LINE_LED_COL_5, 
-  LINE_LED_COL_6, 
-  LINE_LED_COL_7, 
-  LINE_LED_COL_8, 
-  LINE_LED_COL_9, 
+  LINE_LED_COL_1,
```
Comment (kdarkhan) [inline main.c:40]: trailing whitespace cleanup

File: main.c [hunk 3]
```diff
@@ -79,24 +80,29 @@ ioline_t ledRows[NUM_ROW * 4] = {
 
 #define ANIMATION_TIMER_FREQUENCY   60
 
+#define KEY_COUNT                   70
```
Comment (kdarkhan) [inline main.c:83]: saw a TODO for defining this, so I did.

File: main.c [hunk 4]
```diff
@@ -184,73 +193,68 @@ void changeMask(uint8_t mask) {
...
+    ledIntensity = (ledIntensity + 1) % 4;
+    executeProfile();
+}
+
+
 /*
  * Set profile and execute it
  */
-void setProfile(){
+void setProfile() {
```
Comment (kdarkhan) [inline main.c:211]: reformatted these and below functions

**Comment by kdarkhan** [comment]
Corresponding PR in the main repo is [here](https://github.com/OpenAnnePro/qmk_firmware/pull/15/files)

**Comment by Codetector1374** [comment]
please rebase your commit. Thanks

**Comment by kdarkhan** [comment]
> please rebase your commit. Thanks

@Codetector1374, just rebased. Thanks!

**Comment by Codetector1374** [comment]
Thanks. Please try to keep style changes a separate PR from content changes. It is kind of hard to read the diff this way. But good work.

**Pull request merged**